index.js

```diff
@@ -7,10 +7,35 @@ const isWsl = require('is-wsl');
 const isDocker = require('is-docker');
 
 const pAccess = promisify(fs.access);
+const pReadFile = promisify(fs.readFile);
 
 // Path to included `xdg-open`.
 const localXdgOpenPath = path.join(__dirname, 'xdg-open');
 
+/**
+ * Returns the mount point for fixed drives in WSL
+ *
+ * @async
+ * @inner
+ * @returns {string} Mount point
+ */
+const getWslMountPoint = async () => {
+	const confFile = '/etc/wsl.conf';
+
+	if (fs.existsSync(confFile)) {
```

Comment:
Maybe flip the condition and early-exit so the entire function isn't in an `if` block